docs/content/nft-marketplace/building-blocks.md

```diff
@@ -22,14 +22,14 @@ If you are coming from Ethereum, the following list shows corresponding modules
 
 | Resource                            | Ethereum                | Flow                                                                                                  |
 | ----------------------------------- | ----------------------- | ----------------------------------------------------------------------------------------------------- |
-| Smart Contract Programming Language | Solidity                | [Cadence](cadence/)                                                                                   |
+| Smart Contract Programming Language | Solidity                | [Cadence](https://developers.flow.com/cadence)                                                                                   |
 | Fungible Token Standard             | ERC-20                  | [Flow Fungible Token Standard](https://github.com/onflow/flow-ft)                                     |
 | NFT Standard                        | ERC-721/ERC-1155        | [Flow NFT Token Standard](https://github.com/onflow/flow-nft)                                         |
 | NFT Metadata Standard               | ERC-721                 | [Flow NFT Metadata Standard](https://github.com/onflow/flow-nft/#nft-metadata)                        |
-| Native Crypto Currency              | ETH                     | [FLOW](flow-token/)                                                                                   |
-| Stablecoins                         | USDC/DAI/USDT           | [FUSD](fusd/), tUSDT, USDC                                                                            |
-| Wallet Interaction Library          | web3.js                 | [Flow Client Library (FCL)](fcl/)                                                                     |
-| Blockchain Interaction SDK          | web3.js                 | [Flow Client Library (FCL)](fcl/), [Flow Go SDK](flow-go-sdk/), and [many others](flip-fest-winners/) |
+| Native Crypto Currency              | ETH                     | [FLOW](https://docs.onflow.org/flow-token/)                                                                                   |
+| Stablecoins                         | USDC/DAI/USDT           | [FUSD](https://docs.onflow.org/fusd/), tUSDT, USDC                                                                            |
+| Wallet Interaction Library          | web3.js                 | [Flow Client Library (FCL)](https://github.com/onflow/fcl-js)                                                                     |
```

Comment:
`fcl/` should be fine as well. it gets redirected, should only change the broken links.
fcl/
flow-go-sdk//

also, use "https://developers.flow.com" as the root cuz docs.onflow.org gets redirected

Reply:
They still resolve to `flow/nft-marketplace/fcl/` or `flow-go-sdk/`. Is there a place I can change for the redirects?

Reply:
links that don't start with '/' will get prepended with "https://developers.flow.com/"
so `(FCL)](fcl/) ` will resolve to https://developers.flow.com/fcl which does get redirected to tools/fcl-js
but they are not broken so shouldn't be changed in this PR.

Reply:
Is this resolve currently live? I'm just basing off what's happening on https://developers.flow.com/flow/nft-marketplace/building-blocks where the links are not redirected properly.